src/frontend/screens/Track/TrackHeaderRight.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {View, StyleSheet} from 'react-native';
+
+import {IconButton} from '../../sharedComponents/IconButton';
+import {useTrackQuery} from '../../hooks/server/track';
+import {useDeviceInfo} from '../../hooks/server/deviceInfo';
+import {UIActivityIndicator} from 'react-native-indicators';
+import EditIcon from '../../images/Edit.svg';
```

Comment:
not clear at all, but i think to be consistent with what the [observation screen header](https://github.com/digidem/comapeo-mobile/blob/dfc7d9b1d0d48b203fb3d2bf68897932c59c17f1/src/frontend/screens/Observation/ObservationHeaderRight.tsx#L7) is doing, this should be using the component from the `sharedComponents/icon`. 

I noticed the default size of them being very different when navigating between the screens 😅